src/index.d.ts

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import type { DrawOptions, EditOptions, ControlPosition } from 'leaflet';
+
+
+interface EditControlProps {
+  onEdited?: Function;
```

Comment:
agree with @mindflay3r 

could we replace the Function type with something more verbose

tnx